gui/slick/js/new/config_postProcessing.js

```diff
@@ -0,0 +1,2 @@
+$('#config-components').tabs();
```

Comment:
I think we can get rid of all the `.tabs()` calls by using `data-toggle="tab"`.
http://getbootstrap.com/javascript/#tabs